responsemanager/peerresponsemanager/peerresponsesender.go

```diff
@@ -6,20 +6,25 @@ import (
 
 	"github.com/ipfs/go-graphsync/peermanager"
 
-	"github.com/ipld/go-ipld-prime/linking/cid"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
 
 	"github.com/ipfs/go-graphsync/ipldbridge"
 
 	logging "github.com/ipfs/go-log"
 	"github.com/ipld/go-ipld-prime"
 
-	"github.com/ipfs/go-block-format"
+	blocks "github.com/ipfs/go-block-format"
 	"github.com/ipfs/go-graphsync/linktracker"
 	gsmsg "github.com/ipfs/go-graphsync/message"
 	"github.com/ipfs/go-graphsync/responsemanager/responsebuilder"
 	"github.com/libp2p/go-libp2p-core/peer"
 )
 
+const (
+	// max block size is the maximum size for batching blocks in a single payload
+	maxBlockSize = 512 * 1024
```

Comment:
Can this value be configurable, injected by the caller (e.g. Filecoin code) at construction?

Reply:
ATM no -- graphsync doesn't have configurability like this -- but it needs to, and soon. I need to add a variadic options structure to the constructor for this sort of configuration -- seperate PR though.